stdlib/task/src/main/ballerina/src/task/Module.md

```diff
@@ -18,70 +14,40 @@ Below are the two types of configurations that can be used to configure a Task L
 
 The `TimerConfiguration` can be used to configure a task that needs to be executed periodically.
 
-The following example creates a listener, which registers a task with an initial delay of 3000 milliseconds and is executed every 1000 milliseconds for 10 times. The `onTrigger ` resource function is triggered when the clock goes off. The `count` variable is incremented by the task.
+The following code snippet shows how to create a listener, which registers a task with an initial delay of 3000 milliseconds and is executed every 1000 milliseconds for 10 times.
 
 ```ballerina
-import ballerina/log;
-import ballerina/task;
-
-// Task Timer configuration record to configure a Task Listener.
 task:TimerConfiguration timerConfiguration = {
     intervalInMillis: 1000,
     initialDelayInMillis: 3000,
     // Number of recurrences will limit the number of times the timer runs.
     noOfRecurrences: 10
 };
 
-// Initialize the listener using pre defined configurations.
 listener task:Listener timer = new(timerConfiguration);
-
-int count = 0;
-
-// Creating a service on the task Listener.
-service timerService on timer {
-    // This resource triggers when the timer goes off.
-    resource function onTrigger() {
-        log:printInfo("Cleaning up...");
-        log:printInfo(count.toString());
-        count = count + 1;
-    }
-}
```

Comment:
Shall we mention how the service template looks like, something like,
```ballerina
// Creating a service on the task Listener.
service timerService on timer {
    // This resource triggers when the timer goes off.
    resource function onTrigger() {
    }
}
```

Reply:
Added.